scripts/snap/create-snapcraft-yaml.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Ensure the script fails on errors
+set -e
+
+# Check if the correct number of arguments are passed
+if [ "$#" -ne 3 ]; then
+  echo "Usage: $0 <destination_directory> <release_tag> <artifact_path>"
+  exit 1
+fi
+
+# Assign arguments to variables
+DESTINATION_DIR="$1"
+RELEASE_TAG="$2"
+ARTIFACT_PATH="$3"
+
+# Ensure the destination directory exists
+mkdir -p "${DESTINATION_DIR}/snap"
+
+# Extract version from RELEASE_TAG, assuming it's in the format "vX.Y.Z"
+VERSION="${RELEASE_TAG#v}"
+
+# Use the provided ARTIFACT_PATH
+SOURCE="$ARTIFACT_PATH"
+
+# Create the snapcraft.yaml file
+cat > "${DESTINATION_DIR}/snap/snapcraft.yaml" <<EOF
+name: algokit-dev
+version: "$VERSION"
+summary: The AlgoKit tool for developers building on the Algorand network.
```

Comment:
```suggestion
summary: The Algorand AlgoKit CLI is the one-stop shop tool for developers building on the Algorand network.
```

Reply:
The max length of summary is 78 characters.

Reply:
@negar-abbasi Updated to 77 characters